components/LoginPanel.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Button, Card, Form, Input, message } from 'antd';
+import { useRouter } from 'next/router';
+
+interface Props {
+  text: string;
+  isSignUp?: boolean;
+}
+
+export const LoginPanel: React.FC<Props> = ({ text, isSignUp = false }) => {
+  const router = useRouter();
+
+  const onFinishFailed = (errorInfo: any) => {
```

Comment:
nit: typically do not want to use `any`  for any variables. Is there something on the AntD docs that has a type for this? If not, should be okay!

Reply:
they used `any` too so ima just leave it for now xD